js/app.js

```diff
@@ -1,27 +1,42 @@
-import { getProductId } from './auxilliary.js';
-import { toggleFavorite, getProductById } from './products.js';
-import { Cart } from './cart.js';
+import { getProductId } from "./auxilliary.js";
+import { toggleFavorite, getProductById } from "./products.js";
+import { Cart } from "./cart.js";
 
+const fav = document.querySelectorAll(".wish-icon");
 
-const fav = document.querySelectorAll('.wish-icon');
+fav.forEach((wishI) =>
+    wishI.addEventListener("click", (event) => {
+        const id = getProductId(event.target);
+        const isFavProduct = toggleFavorite(id);
 
-fav.forEach(wishI => wishI.addEventListener('click', (event) => {
-    const id = getProductId(event.target);
-    const isFavProduct = toggleFavorite(id);
-
-    event.target.classList.toggle("fa-heart", isFavProduct)
-    event.target.classList.toggle("fa-heart-o", !isFavProduct)
-
-}));
+        event.target.classList.toggle("fa-heart", isFavProduct);
+        event.target.classList.toggle("fa-heart-o", !isFavProduct);
+    })
+);
 
-const btns = document.querySelectorAll('.btn');
+const cartElement = document.querySelector(".cart-element");
+const btns = document.querySelectorAll(".btn");
 const cart = new Cart();
-btns.forEach((button)=>{
-    button.addEventListener('click',(e)=>{
-        e.preventDefault()
+btns.forEach((button) => {
+    button.addEventListener("click", (e) => {
+        e.preventDefault();
         const id = getProductId(e.target);
-        cart.addToCart(id)
-        console.log(cart)
-        console.log(getProductById(id))
-    })
-})
\ No newline at end of file
+        let product = getProductById(id);
+        cart.addToCart(id);
+
+        console.log(cart);
+        console.log(product);
+        console.log(cartElement);
+
+        cartElement.innerHTML += `
```

Comment:
Majd azt is old meg, hogy ne tobbszor keruljon be ugyan az az elem, hanem csak egyszer nagyobb `qty`-val

Reply:
potosan azon gondolkozom fél órája :D de nem jön az ihlet